app/controllers/AdditionalMetadataSummaryController.scala

```diff
@@ -23,9 +25,20 @@ class AdditionalMetadataSummaryController @Inject ()(val consignmentService: Con
       consignment <- consignmentService.getConsignmentFileMetadata(consignmentId, request.token.bearerAccessToken, selectedFileIds)
       response <- consignment.files match {
         case first :: _ =>
-          Future(Ok(views.html.standard.additionalMetadataSummary(fileName, consignment.consignmentReference, first.metadata, request.token.name)))
+          Future(Ok(views.html.standard.additionalMetadataSummary(fileName, consignment.consignmentReference, getMetadata(first.metadata), request.token.name)))
         case Nil => Future.failed(new IllegalStateException(s"Can't find selected files for the consignment $consignmentId"))
       }
     } yield response
   }
+
+  private def getMetadata(metaData: GetConsignment.Files.Metadata): Map[String, String] = {
```

Comment:
This is a suggestion so feel free to ignore me but would it be better to have another case class here with `String` for each of these attributes rather than the Map. Then the additionalMetadataSummary view can use it as well. What do you think?

Reply:
Yeah, good suggestion. I have updated this.